contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -5447,14 +5447,15 @@ bbf_ExecDropStmt(DropStmt *stmt)
 	{
 		foreach(cell, stmt->objects)
 		{
+			relation = NULL;
```

Comment:
When case is schema: We only have one object
Since DROP SCEHMA allows only one entry.

For procedure, functions, will need changes

Reply:
made changes at two places
1st one covers case for table, view, sequence
2nd one covers procedure, function, type

for schema no change is needed since only one schema is allowed to be dropped at a time